package.json

```diff
@@ -8,15 +8,17 @@
   "description": "Updates Volta config in package.json so Node match's what your NW.js version uses",
   "scripts": {
     "lint": "eslint --rulesdir=./node_modules/eslint-config-tjw-jest/custom-eslint-rules --ext .js --config=.eslintrc.cjs .eslintrc.cjs",
-    "fix": "eslint --fix --rulesdir=./node_modules/eslint-config-tjw-jest/custom-eslint-rules --ext .js --config=.eslintrc.cjs .eslintrc.cjs"
+    "fix": "eslint --fix --rulesdir=./node_modules/eslint-config-tjw-jest/custom-eslint-rules --ext .js --config=.eslintrc.cjs .eslintrc.cjs",
+    "test": "vitest run --pool=forks"
```

Comment:
Trying to run `npm t` locally and getting errors, don't merge yet.

Reply:
There was an optional dep missing and the instructions were to delete the package-lock and node_modules and do a fresh install. So I did, and the package lock has thousands of lines changed now, but the tests are still failing.

Reply:
I got it working locally, and made a bunch of other changes too, too many to just push up directly into this PR. So I made a separate PR against this one for you to review. I think with it merged into this one, it would be ready for this to then be merged into `main`. Marked you as the reviewer on the other one.

Reply:
Changes have been merged in now.